source/Calamari.Tests/Fixtures/Substitutions/Approved/SubstitutionsFixture.ShouldApplyFiltersDuringSubstitution.approved.txt

```diff
@@ -0,0 +1,9 @@
+JsonEscape: =:'\"\\\r\n\t <>

```

Comment:
The current JSON behaviour is actually not right. This should be escaping newlines. I'm going to raise a PR on Octostache to fix this.
```suggestion
```